.nvmrc

```diff
@@ -0,0 +1 @@
+22.16
```

Comment:
This is the fourth different version across our main repos (mattermost, desktop, mobile, now load-test). Surely one day we'll find consistency :sweat_smile: